production/helm/loki/templates/distributor/deployment-distributor.yaml

```diff
@@ -44,6 +44,12 @@ spec:
         {{- toYaml . | nindent 8 }}
         {{- end }}
     spec:
+      {{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
can we please use `KubeVersion` instead? `GitVersion` appears to be [deprecated](https://github.com/helm/helm/blob/main/pkg/chartutil/capabilities.go#L83) and really confused me, haha